desc/grid.py

```diff
@@ -416,11 +418,6 @@ def _create_nodes(
             node spacing, based on local volume around the node
 
         """
-        self._L = L
```

Comment:
any reason for getting rid of these assignments?

Reply:
They are already set in the constructor. The input for `L, M, N, NFP` in this function is s`elf.L, self.M, self.N, self.NFP` anyway.

Reply:
ok cool